src/main/java/org/gridsuite/study/server/dto/InfoTypeParameters.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.study.server.dto;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Data
+@Builder
+public class InfoTypeParameters {
+    public static final String QUERY_PARAM_DC_POWERFACTOR = "dcPowerFactor";
+    public static final String QUERY_PARAM_OPERATION = "operation";
+    private String infoType;
+    private Map<String, String> optionalParameters;
```

Comment:
I find that this way of putting together these two data -which are not even the only parameters- quite artificial. The resulting `InfoTypeParameters` is a class name that looks more like a collage than a real data structure.